src/components/qas/questions/qaCompoundTextInput/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Card from '../../../site/card';
+import QATextInput from '../qaTextInput';
+import styles from './styles.module.scss';
+
+class QACompoundTextInput extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hasFocus: false,
+    };
+  }
+
+  updateActive = hasFocus => {
+    this.setState(prevState => ({
+      ...prevState,
+      hasFocus,
+    }));
+  };
+
+  render() {
+    const { questions, activeId, answers, handleAnswerSelect } = this.props;
+    const { hasFocus } = this.state;
+    const cardClasses = classnames('qa-card', { active: hasFocus });
+    const containerClasses = classnames('qa-wrapper');
+
+    return (
+      <Card className={cardClasses}>
+        <div className={containerClasses}>
```

Comment:
don't really need this containerClass variable or classnames if you're only giving it a static classname